### PR TITLE
Add Xperia Z2 Tablet WiFi

### PR DESCRIPTION
**Pull request by opendata26**
Needs #257 reverted for the adapter to show up.
Has working ssh, virtual terminal, drm, framebuffer and  #weston.

See #338

### CODE REVIEW

File: aports/device/device-sony-castor_windy/initfs-hook.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+# set framebuffer resolution
```
Comment (ollieparanoid) [inline aports/device/device-sony-castor_windy/initfs-hook.sh]: Is this empty hook intentional? If you don't need it, please remove it (remove this file, remove it from the APKBUILD's sources and remove the install line in the `package()` function).

Comment (opendata26) [inline aports/device/device-sony-castor_windy/initfs-hook.sh]: Nope, will remove it

**Comment by ollieparanoid** [comment]
Could you rebase to master and check if all network issues are gone?
...and sorry that I just see this now, please rename the aports to use `-` instead of `_` as per our [package naming convention](https://github.com/postmarketOS/pmbootstrap/wiki/Package-naming-convention).
I hope we can merge this soon, thanks for all your hard work on this!

**Comment by MartijnBraam** [comment]
Looks good to merge now, Thanks

**Pull request merged**